### PR TITLE
Extract call accessory, fix alignment when no accessory is shown

### PR DESCRIPTION
**Pull request by daehn**
## What's new in this PR?

* Extract call accessory component into separate view controller.
* Fix alignment when no accessory is shown.

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/Calling/CallInfoViewController/CallInfoViewController.swift [hunk 1]
```diff
@@ -98,35 +97,31 @@ final class CallInfoViewController: UIViewController, CallActionsViewDelegate {
...
+    // MARK: - Actions + Delegates
 
     func callActionsView(_ callActionsView: CallActionsView, perform action: CallAction) {
         Calling.log.debug("\(action) button tapped")
         delegate?.infoViewController(self, perform: action)
     }
+    
+    func callParticipantsViewControllerDidSelectShowMore(viewController: CallParticipantsViewController) {
+        Calling.log.debug("Show more participants tapped")
+        // TODO: Do we need this or should we directly push the new controller?
```
Comment (typfel) [inline Wire-iOS/Sources/UserInterface/Calling/CallInfoViewController/CallInfoViewController.swift]: Can't it be considered internal state of the `CallInfoViewController`

Comment (daehn) [reply Wire-iOS/Sources/UserInterface/Calling/CallInfoViewController/CallInfoViewController.swift]: Yeah, I think so. We can just create + push the detail vc in here.

Comment (daehn) [reply Wire-iOS/Sources/UserInterface/Calling/CallInfoViewController/CallInfoViewController.swift]: Ah no, I think that `CallViewController` should be responsible for that, as that one will be the one presented inside a `UINavigationController`.

Comment (typfel) [reply Wire-iOS/Sources/UserInterface/Calling/CallInfoViewController/CallInfoViewController.swift]: Ok

**Pull request merged**